### PR TITLE
upload-video

### PR DESCRIPTION
**Pull request by ShakoHo**
@askeing r?

### CODE REVIEW

File: lib/common/pyDriveUtil.py [hunk 1]
```diff
@@ -59,13 +68,23 @@ def main():
                             help='Specify file name your want to update.', required=True)
     arg_parser.add_argument('-c', action='store', dest='input_content', default=None,
                             help='Specify content want to update.', required=True)
```
Comment (askeing) [inline lib/common/pyDriveUtil.py]: maybe the `input_folder_uri`, `input_file_name`, and `input_content` can set as positional arguments.
so we can skip `-u` `-f` `-c` :P
ex:
```python
     arg_parser.add_argument('Folder_URI', default=None,
                             help='Specify the uri path of folder.', required=True)
     arg_parser.add_argument('Filename', default=None,
                              help='Specify file name your want to update.', required=True)
     arg_parser.add_argument('Content', default=None,
                              help='Specify content want to update.', required=True)
```

File: lib/helper/uploadAgentHelper.py [hunk 2]
```diff
@@ -1,13 +1,21 @@
...
 import platform
 import desktopHelper
+import videoHelper
 from datetime import date
+from ..common.pyDriveUtil import PyDriveUtil
 from desktopHelper import DEFAULT_BROWSER_TYPE_FIREFOX
 from desktopHelper import DEFAULT_BROWSER_TYPE_CHROME
 
+DEFAULT_UPLOAD_VIDEO_YAML_SETTING = "./mozhasalvideo.yaml"
+DEFAULT_UPLOAD_VIDEO_MYCRED_TXT = "./mycreds_mozhasalvideo.txt"
```
Comment (askeing) [inline lib/helper/uploadAgentHelper.py]: do we have any instructions about how to setup setting and cred?

**Comment by askeing** [comment]
lgtm

File: lib/helper/uploadAgentHelper.py (hunk 2 above)
Comment (ShakoHo) [inline lib/helper/uploadAgentHelper.py]: We just need to put those two files in working dir, and those two credential are generated based on our test google account

**Pull request merged**